upgrade_test.py

```diff
@@ -1009,27 +1010,26 @@ def test_kubernetes_operator_upgrade(self):
             stress_cmd=self._cs_add_node_flag(self.params.get('stress_cmd_r'))))
 
     def test_kubernetes_platform_upgrade(self):
+        step = itertools_count(start=1)
         self.k8s_cluster.check_scylla_cluster_sa_annotations()
 
-        InfoEvent(message='Step1 - Populate DB with data').publish()
+        InfoEvent(message=f"Step{next(step)} - Populate DB with data").publish()
```

Comment:
I think `Step{next(step)}` hinder a bit the usability when trying to investigate, whom not familiar with this specific change, would  look for what he sees in the log, and won't find it in the code.

Reply:
Ok, will change.